### PR TITLE
Change wording on ADOT Section

### PR DESCRIPTION
**Pull request by willarmiros**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### :mega: OpenTelemetry Java with AWS X-Ray
 
-AWS X-Ray recommends using AWS Distro for OpenTelemetry (ADOT) to instrument your application **instead of this X-Ray SDK** due to its wider range of features and instrumentations. See the [AWS X-Ray docs on Working with Java](https://docs.aws.amazon.com/xray/latest/devguide/xray-java.html) for more help with choosing between ADOT and X-Ray SDK.
+AWS X-Ray supports using OpenTelemetry Java and the AWS Distro for OpenTelemetry (ADOT) Collector to instrument your application and send trace data to X-Ray. The OpenTelemetry SDKs are an industry-wide standard for tracing instrumentation that provide more instrumentations and have a larger community for support, but may not have complete feature parity with the X-Ray SDKs. See the [AWS X-Ray documentation on Working with Java](https://docs.aws.amazon.com/xray/latest/devguide/xray-java.html) for more help with choosing between ADOT and X-Ray SDK.
```
Comment (carolabadeer) [inline README.md:5]: Do we have any public docs on feature parity that we can also link here?

Comment (willarmiros) [reply README.md:5]: We link this doc right now: https://docs.aws.amazon.com/xray/latest/devguide/xray-java.html

Which in turn links to https://docs.aws.amazon.com/xray/latest/devguide/xray-instrumenting-your-app.html#xray-instrumenting-choosing, which contains feature parity info. However taking a second look, it's probably best to link directly to this page.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ### :mega: OpenTelemetry Java with AWS X-Ray
 
-AWS X-Ray recommends using AWS Distro for OpenTelemetry (ADOT) to instrument your application **instead of this X-Ray SDK** due to its wider range of features and instrumentations. See the [AWS X-Ray docs on Working with Java](https://docs.aws.amazon.com/xray/latest/devguide/xray-java.html) for more help with choosing between ADOT and X-Ray SDK.
+AWS X-Ray supports using OpenTelemetry Java and the AWS Distro for OpenTelemetry (ADOT) Collector to instrument your application and send trace data to X-Ray. The OpenTelemetry SDKs are an industry-wide standard for tracing instrumentation that provide more instrumentations and have a larger community for support, but may not have complete feature parity with the X-Ray SDKs. See [choosing between the ADOT and X-Ray SDKs](https://docs.aws.amazon.com/xray/latest/devguide/xray-instrumenting-your-app.html#xray-instrumenting-choosing) for more help with choosing between the two.
```
Comment (atshaw43) [inline README.md:5]: "The OpenTelemetry SDKs are an industry-wide standard for tracing instrumentation that provide more instrumentations and have a larger community for support, but may not have complete feature parity with the X-Ray SDKs"

Nit: This feels like a run-on sentence.

Suggestion:
The OpenTelemetry SDKs are an industry-wide standard for tracing instrumentation. They provide more instrumentations and have a larger community for support, but may not have complete feature parity with the X-Ray SDKs

**Comment by willarmiros** [comment]
@atshaw43 @carolabadeer PTAL

**Review by carolabadeer — APPROVED** [review]
LGTM!

**Review by atshaw43 — APPROVED** [review]

**Pull request merged**